### ISSUE TITLE
Add option "use-utc" to set dates in UTC to align with upstream

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#1324]
There's an incoming change in AsciiDoc Python to add a "use-utc" option that will set dates in UTC instead of local time. We should do the same in Asciidoctor. See https://github.com/asciidoc/asciidoc/pull/70.

**Comment by mojavelinux** [comment]
Other ideas for the option name:

* use-utc-dates
* utc-dates
* utc

Any examples of existing programs with a similar option?

**Comment by michael-o** [comment]
`use-utc-timezone` seems to be the most precise one even it is long. Better yet -- `:timezone: Europe/Berlin` or `:timezone: Etc/Utc` which basically is the timezone ID from Olson database.

**Comment by mojavelinux** [comment]
I kind of like the idea of setting the timezone explicitly. That way, we get a simpler attribute name and much more control. This could be essential for integration with other tools as well. I like it!

**Comment by mojavelinux** [comment]
I'd be happy to accept the value of "UTC", as well as time offsets (e.g., +02:00). That would avoid the need for an external dependency. If you want to use another identifier, then we have to load the tzinfo library.

**Comment by michael-o** [comment]
@mojavelinux Working with timezone offsets is a constant source of error. They should be avoided at all times. Why? DST changes, timezone shifts, etc and you don't even notice that. Zone IDs are the safest bet. I don't know how this works in Ruby but it is straight forward in Java and Oracle database.

**Comment by mojavelinux** [comment]
> Working with timezone offsets is a constant source of error. 

I agree. I was just pointing out that if a timezone offset is specified, we don't need to load the tz database (perhaps because a tool already resolved it and passed the value to Asciidoctor, we don't know). We also don't need to load it for UTC since it is fixed. So those were merely implementation comments.

**Comment by michael-o** [comment]
@mojavelinux You are probably talking about [this](https://rubygems.org/gems/tzinfo) gem? I agree that UTC is fixed can be supported right away. Anyway, zone IDs should be encouraged and offsets discouraged.

**Comment by mojavelinux** [comment]
$subject can be accomplished by invoking asciidoctor as follows:

```
TZ=UTC asciidoctor input.adoc
```
